test_data/components/basic/optional_params/OWNERS

```diff
@@ -0,0 +1,5 @@
+approvers:
+  - test-team
```

Comment:
Let's use real OWNERS from the root of the repo to avoid unnecessarily granting Prow access to the repo.

Reply:
updated: copied root OWNERS over top of test_data mock OWNERS files, adjusted mock READMEs accordingly